examples/cron/evm-deploy-watcher/Makefile

```diff
@@ -0,0 +1,7 @@
+build:
+	@cargo build --lib --target wasm32-unknown-unknown
+	@cosmwasm-check target/wasm32-unknown-unknown/debug/evm_deploy_watcher.wasm
+
+build-release:
+	@cargo build --lib --target wasm32-unknown-unknown --release
+	@cosmwasm-check target/wasm32-unknown-unknown/release/evm_deploy_watcher.wasm
```

Comment:
This check performs static check => ensures the produced binary will be compatible with cosmwasm before any deployments